lib/fog/google/models/compute/server.rb

```diff
@@ -206,6 +206,32 @@ def add_ssh_key username, key
           return self.metadata
         end
 
+        def map_service_accounts(scope_array)
+          gce_scope_aliases = {
```

Comment:
`gce_scope_aliases` seems like it should be a constant, declared in the class scope or somewhere else.